packages/core/src/components/DatePicker/DatePicker.tsx

```diff
@@ -116,6 +116,7 @@ const DatePicker: VibeComponent<DatePickerProps, HTMLElement> = forwardRef<HTMLD
 
     const onDateRangeChange = useCallback(
       (date: RangeDate) => {
+        if (!onPickDate) return;
```

Comment:
Not exactly sure what is the library convention, but i believe in this way it is more clear.
Another option is run the onPickDate?.(....)
```suggestion
        if (!onPickDate) { 
            return;
        }
```